### PR TITLE
Adding Enter key functionality to search events on homepage

### LINKED ISSUE #1893: Adding Enter key functionality to search events on homapage
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sk9331657**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adding Enter key functionality to search events on homepage

#### Changes proposed in this pull request:

-Adding Enter key functionality to search events on homepage
-
-

Fixes #1893

### CODE REVIEW

File: app/components/quick-filter.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ import Component from '@ember/component';
 import moment from 'moment';
 
 export default Component.extend({
+  didInsertElement: () => {
```
Comment (CosmicCoder96) [inline app/components/quick-filter.js]: This is not the ember way, either use a form element to trigger an event or semantic UI callbacks, don't use jquery and don't add any additional selectors like `id='searchevents'.

Comment (sk9331657) [reply app/components/quick-filter.js]: ok

**Comment by sk9331657** [comment]
@CosmicCoder96  Done.

File: app/components/quick-filter.js [hunk 2]
```diff
@@ -69,6 +67,12 @@ export default Component.extend({
       this.set('disableClear', false);
     },
 
+    enter() {
```
Comment (CosmicCoder96) [inline app/components/quick-filter.js]: rename this action to `keyPressed`

Comment (sk9331657) [reply app/components/quick-filter.js]: OK

File: app/utils/dictionary/payment.js [hunk 3]
```diff
@@ -176,4 +176,4 @@ export const paymentCurrencies = [
     name   : 'Swiss franc',
     stripe : true
   }
-];
+];
```
Comment (CosmicCoder96) [inline app/utils/dictionary/payment.js]: @sk9331657  Remove this change, please.

Comment (sk9331657) [reply app/utils/dictionary/payment.js]: Done

**Comment by sk9331657** [comment]
> Commit message does not make sense. How is dictionary related to this issue or PR?
> 
> Also do not add issue number in the PR title. It's of no use.

Resolved .

**Pull request merged**